kernel_hardening_checker/checks.py

```diff
@@ -673,6 +673,19 @@ def add_cmdline_checks(l: List[ChecklistObjType], arch: str) -> None:
     if arch in ('X86_64', 'X86_32'):
         l += [CmdlineCheck('self_protection', 'clipos', 'iommu', 'force')]
 
+    # 'self_protection', 'a13xp0p0v'
+    l += [OR(CmdlineCheck('self_protection', 'a13xp0p0v', 'lockdown', 'confidentiality'),
```

Comment:
This check and other checks you've added are inspired by the KSPP kconfig recommendations.

- Please use the `kspp` decision and move them in the section above.

- Please add comments similar to `# consequence of 'slab_nomerge' by kspp` (see in the section above).